frontend/utils/touchInteractions.js

```diff
@@ -0,0 +1,4 @@
+import Vue from 'vue'
+import Vue2TouchEvents from 'vue2-touch-events'
```

Comment:
Can you either:

- add a TODO comment here to load this only if touch events are supported
- implement this PR dynamically loading this functionality (using `@import`) only if touch events are supported on this device